tests/Unit/ChurnPHPApplicationTest.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+
+namespace Churn\Tests\Unit;
+
+use Churn\ChurnPHPApplication;
+use Churn\Tests\BaseTestCase;
+
+class ChurnPHPApplicationTest extends BaseTestCase
+{
+    const SEMANTIC_VERSIONING_PATTERN = '\d+\.\d+.\d+';
+
+    /** @test **/
```

Comment:
I'm nitpicking here but it should be:
```
    /** @test */
```

See: https://docs.phpdoc.org/latest/guides/docblocks.html

Reply:
Thank you for point it out!

I've applied the fix through the test folder, please see c9b1ef3.